docs/beta/grain.md

```diff
@@ -1,30 +1,44 @@
 ---
 title: How Grain Works
+description: ""
 image: img/uploads/grain-token-icon.png
 ---
 ## 🌾 What is Grain?
 
-Grain is a community-specific digital currency which is issued on the basis of [Cred](/docs/beta/cred) scores. Grain represents how much a participant has supported that community, either through contributions or financial support. Grain can be used to pay contributors, or for “Boosting” contributions or initiatives. 
+Grain is a community-specific digital currency which is issued on the basis of [Cred](/docs/beta/cred) scores. Grain represents how much a participant has supported that community, either through contributions or financial support. Grain can be used to pay participants, or for “Boosting” contributions or initiatives. 
 
 Boosting allows a participant to burn Grain in order to increase the Cred of a specific contribution. Boosting acts as a way of helping to set the project’s values and priorities, and gives the Booster an increased say in the direction of the project. When you use Grain for Boosting, you get a share of the Cred for whatever you are Boosting, which in turn will earn you future Grain.
 
-Projects can choose to issue their Grain as a cryptocurrency or token, for example by issuing Grain as an ERC-20 token on Ethereum. This allows Grain to be a conduit with which a project can pay or financially reward contributors.
+Projects can choose to issue their Grain as a cryptocurrency or token, for example by issuing Grain as an ERC-20 token on Ethereum. This allows Grain to be a conduit with which a project can pay or financially reward participants.
 
 Note that Grain does not _have_ to be tied to any actual monetary concept, such as cryptocurrency. A SourceCred project can name it “Points” and use it solely as a scoring system in addition to Cred values.
 
 ## How you earn Grain
 
-Grain is created during what we call “Harvests.” A Harvest is a periodic cycle in which the project produces new Grain tokens, and distributes those to contributors based on their Cred scores. Your Cred score determines the percentage of Grain you receive out of the total amount allocated to the community. 
+Grain is created during what we call “Harvests.” A Harvest is a two-fold process with periodic cycles, which can be configured independently:
```

Comment:
```suggestion
Grain is created during what we call “Harvests.” A Harvest is a two-step process in which the Cred is recomputed (so scores are as fresh as possible), and Grain is then distributed based on those Cred scores.
```

"A two-fold process with periodic cycles" is pretty confusing, and suggests that an individual harvest contains multiple cycles.

Reply:
Good suggested changes! (I mean, an individual harvest _does_ contain multiple cycles: 4x7 Cred Calculations. Buuuut you're right that that adds confusion, not clarity 😂)